### DISCUSSION TITLE
ArchLinux.org

### DISCUSSION THREAD

**Discussion by Covkie** [catppuccin/catppuccin#2206] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://userstyles.world/style/1860/arch-linux-dark-theme

I found this while googling: https://gist.github.com/andreasgrafen/7d2b0753e27208cc6feca48b5a29b845

### Any additional comments?

the gorb :3 distro website. (specifically the wiki)

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
https://github.com/catppuccin/userstyles/tree/main/styles/arch-wiki exists now.
